### PR TITLE
Add support for local storage permission

### PR DESCRIPTION
**Pull request by Amejia481**
We need the new ac version that contains the support for the local storage permission

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by Amejia481** [comment]
As https://github.com/mozilla-mobile/android-components/issues/3153 landed, this should be ready to land

File: app/src/main/res/drawable/ic_storage.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24"
+    android:autoMirrored="true">
+  <path
+      android:fillColor="?primaryText"
+      android:pathData="M2,20h20v-4L2,16v4zM4,17h2v2L4,19v-2zM2,4v4h20L22,4L2,4zM6,7L4,7L4,5h2v2zM2,14h20v-4L2,10v4zM4,11h2v2L4,13v-2z"/>
+</vector>
```
Comment (pocmo) [inline app/src/main/res/drawable/ic_storage.xml:13]: Oh, we need to duplicate this? :(

**Review by pocmo — APPROVED** [review]

File: app/src/main/res/drawable/ic_storage.xml (hunk 1 above)
Comment (Amejia481) [reply app/src/main/res/drawable/ic_storage.xml:13]: No really, I put it as place holder for the animated version, similar to [other permissions icon](https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/res/drawable/ic_location.xml), I was trying to create a strike through version, but  I'm not that good with designing tools 🦄. I followed @NotWoods 's [blog](https://medium.com/firefox-mobile-engineering/animating-a-strike-through-on-android-with-animated-vector-drawable-and-animatedstatelistdrawable-a77e66f9790f) but I was spending a lot time with [Shape Shifter](https://shapeshifter.design/), without much progress :(

**Comment by jonalmeida** [comment]
We can close this now that we've cherry-picked these changes as part of https://github.com/mozilla-mobile/fenix/pull/15786.

**Pull request closed without merging**